solidity/test/BancorConverter.js

```diff
@@ -5,24 +5,23 @@ const fs = require('fs');
 
 const utils = require('./helpers/Utils');
 const ContractRegistryClient = require('./helpers/ContractRegistryClient');
-
```

Comment:
@yudilevi : This space is there on purpose (and if you still think that we should remove it, then it appears in a bunch of other files).

Reply:
I don't think there should be any space between imports

Reply:
@yudilevi:
This space separates between normal `require` statements and `artifacts.require` statements.
If you still think it's unnecessary, then we should probably remove it also in all the other files.

Reply:
yeah, I don't recall any popular linter that adds spaces between imports